### ISSUE TITLE
[Question] clearByName Android alternative?

### ISSUE THREAD

**Issue by francisleigh** [react-native-cookies/cookies#100]
Hello! 
I am looking to be able to clear all Cookies for a specific Domain and can see that `clearByName` is only supported for IOS. 

e.g docs
```javascript
// clear cookies with name (IOS ONLY)
CookieManager.clearByName('http://example.com', 'cookie name', useWebKit)
	.then((succcess) => {
		console.log('CookieManager.clearByName from webkit-view =>', succcess);
});
```

Has anyone found a way to do this with Android? 

Thank you
Francis

**Comment by FrancisLeigh** [comment]
Thinking [this package](https://github.com/shimohq/react-native-cookie) could be a viable alternative to this library.

**Comment by trithien98** [comment]
Need this property for the android platform. I don't want to clearAll cookies

**Comment by safaiyeh** [comment]
Yeah Android doesn't support this out of the box https://developer.android.com/reference/android/webkit/CookieManager, hence only supporting iOS.

Open to seeing how we could design this feature since its been highly requested.

**Comment by clemwo** [comment]
Any update on this issue?  
I'd love to know how to clear cookies for a certain URL on android

**Comment by clemwo** [comment]
Ok, so what works for me is simply setting the value of the cookie I want to delete to an empty string. 
But I honestly have no clue why the token gets automatically deleted when I set the value to an empty string.
Can anyone else try this out and confirm this?  
@safaiyeh 

```typescript
CookieManager.set(<url>, { name: "<cookieName>", value: ""})
```

Edit. In [RFC 6265](https://datatracker.ietf.org/doc/html/rfc6265) it also says that in order to delete cookies, servers can simply send a cookie with the same value but an expiry date in the past. Maybe that's what's happening here? Either way that also seems like a good way to delete a single cookie
4.1.2
> Notice that servers can delete cookies by sending the user agent a
   new cookie with an Expires attribute with a value in the past.
   
 Edit2. Did another test. 
This deletes the cookie with the value of cookieName:

```
    CookieManager.set(url, {
      name: cookieName,
      value: '',
      expires: '2024-05-30T12:30:00.00-05:00',
    })
```
obviously the expiry date is not needed for the deletion here

**Comment by safaiyeh** [comment]
This project has been deprecated and archived. Please migrate to:
- [`@preeternal/react-native-cookie-manager`](https://github.com/Preeternal/react-native-cookie-manager)
- [`react-native-nitro-cookies`](https://github.com/l2hyunwoo/react-native-nitro-cookies)

Thank you for your contributions! ❤️
